controllers/apply/under10k/fields/your-idea.js

```diff
@@ -100,7 +99,7 @@ module.exports = {
         const maxWords = 150;
 
         function prioritiesCOVID19() {
-            if (flags.enableGovCOVIDUpdates) {
+            if (flags.enableGovCOVIDUpdates && projectCountry === 'england') {
```

Comment:
This the key change adding a `projectCountry === 'england'` check for this version of the priorities.